skiko/src/jsMain/kotlin/org/jetbrains/skiko/SkiaLayer.js.kt

```diff
@@ -0,0 +1,64 @@
+package org.jetbrains.skiko
+
+import kotlinx.browser.window
+import org.jetbrains.skiko.w3c.HTMLCanvasElement
+import org.w3c.dom.events.KeyboardEvent
+import org.w3c.dom.events.MouseEvent
+import org.w3c.dom.events.WheelEvent
+
+internal actual fun SkiaLayer.bindCanvasEventsToSkikoView(
+    canvas: HTMLCanvasElement
+) {
+    val htmlCanvas = canvas.unsafeCast<org.w3c.dom.HTMLCanvasElement>()
+
+    htmlCanvas.addEventListener("mousedown", { event ->
+        event as MouseEvent
```

Comment:
We can have helper extenstion function like Element.addMouseEventListener,  addKeyBoardEventListener
or event just generic version of it so that we won't have to case in every handler  (just scrolled down and realised that his was just moved so anyway any potential changes are not supposed to be in the framework of this particular PR, so nevermind)